critiquebrainz/db/exceptions.py

```diff
@@ -11,3 +11,7 @@ class NoDataFoundException(DatabaseException):
 class BadDataException(DatabaseException):
     """Should be used when incorrect data is being submitted."""
     pass
+
+class IntegrityError(DatabaseException):
```

Comment:
Description for `BadDataException` says "Should be used when incorrect data is being submitted." To me that seems like exactly the case you use this new exception for. So that addition seems unnecessary.